### PR TITLE
1549 by Inlead: Implement customerror module.

### PR DESCRIPTION
**Pull request by inlead**
http://platform.dandigbib.org/issues/1549

### CODE REVIEW

**Comment by JrgenGNielsen** [comment]
Please fix the merge conflicts - nice otherwise

File: ding2.info [hunk 1]
```diff
@@ -27,6 +27,8 @@ dependencies[] = pagepreview
 dependencies[] = admin_menu
 dependencies[] = admin_menu_toolbar
 dependencies[] = toolbar
+dependencies[] = customerror
+dependencies[] = customerroralt
```
Comment (kasperg) [inline ding2.info]: Why are there two dependencies here?

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Pull request merged**